app/models/warehouse/case_report.rb

```diff
@@ -163,6 +163,17 @@ def dequote_and_truncate(text)
       def humanize_boolean(boolean)
         boolean ? 'Yes' : nil
       end
+
+      private 
+      def get_requester_type(kase)
+        if kase.offender_sar?
+          kase.third_party? ? kase.third_party_relationship : 'data subject'
```

Comment:
Tiny thing, but could we get rid of this check for offender_sar? 

I think if we added a `requester_type` method to the model offender.rb we could put line 51 back how it was